README.md

```diff
@@ -48,7 +48,7 @@ After downloading the two files, use the following command to convert your priva
 $ openssl ec -in apikey.key -outform PEM -out apikey.key
 ```
 
-Then copy (rename as follows) your TLS certificate and PEM private key file into your py-common installation path as follows. You should mark down the absolute directory of your cert path `./py_common-1.5.0-py2.7.egg/cryption/ecc/certs`, because this path is an essential parameter to create a wallet client.
+Then copy (rename as follows) your TLS certificate and PEM private key file into your py-common installation path as follows. Please pay special attention to the absolute directory of your cert path `./py_common-1.5.0-py2.7.egg/cryption/ecc/certs`, because this path is an essential parameter to create a wallet client.
```

Comment:
1. There is no such saying as absolute directory, only absolute path;
2. We could abbreviate certificate to cert in the code, but in documents it should be certificate;
3. because ... is way too Chinese style.

Reply:
fixed